opm/simulators/wells/MultisegmentWellSegments.hpp

```diff
@@ -128,7 +128,7 @@ class MultisegmentWellSegments
 
     Scalar perforation_depth_diff(const int perf) const
     {
-        return perforation_depth_diffs_[perf];
+        return global_perforation_depth_diffs_[perf];
```

Comment:
This looks really dangerous. As a developer I cannot tell that this needs global perforations